prime-dotnet-webapi/Migrations/20210614235905_AddSiteStatusHistory.cs

```diff
@@ -44,6 +37,30 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 name: "IX_SiteStatus_SiteId",
                 table: "SiteStatus",
                 column: "SiteId");
+
+
+            // Migrate the site status data to the new SiteStatus table
+            // bump old site status by 1, and insert into new table
+            migrationBuilder.Sql(
+                    "INSERT INTO public.\"SiteStatus\" (\"CreatedUserId\", \"CreatedTimeStamp\", \"UpdatedUserId\", \"UpdatedTimeStamp\", \"SiteId\", \"StatusType\", \"StatusDate\")"
+                    + " SELECT \"CreatedUserId\", \"CreatedTimeStamp\", \"UpdatedUserId\", \"UpdatedTimeStamp\", \"Id\" AS \"SiteId\", (\"Status\" + 1) AS \"StatusType\", \"UpdatedTimeStamp\" AS \"StatusDate\""
+                    + " FROM public.\"Site\""
+                    + " WHERE \"SubmittedDate\" IS NOT NULL"
+            );
+
+            // the old status 1 without SubmittedDate becomes the new Active
+            migrationBuilder.Sql(
+                  "INSERT INTO public.\"SiteStatus\" (\"CreatedUserId\", \"CreatedTimeStamp\", \"UpdatedUserId\", \"UpdatedTimeStamp\", \"SiteId\", \"StatusType\", \"StatusDate\")"
+                + " SELECT \"CreatedUserId\", \"CreatedTimeStamp\", \"UpdatedUserId\", \"UpdatedTimeStamp\", \"Id\" AS \"SiteId\", 1 AS \"StatusType\", \"UpdatedTimeStamp\" AS \"StatusDate\""
+                + " FROM public.\"Site\""
+                + " WHERE \"Status\" = 1 AND \"SubmittedDate\" IS NULL"
```

Comment:
All of our existing examples of calling bare SQL have been cleaned up from the repo, unfortunately. If you would like to see some examples, try checking out any commit on develop before PR#1367 (such as 990d624211a22a5eb3b8ca0e2ca98b85ca96e5f2) and looking at 20200609020423_UpdateVendorTableNameAndVendorIdToCode.cs. 

I have two suggestions for how to format these SQL queries
1. Please use the string literal syntax `@""`. Among other reasons, it means you do not have to use the '+'s to string concatenate these strings across multiple lines.
2. Please make sure you terminate your SQL with a semicolon. This code might run fine in DBeaver but may cause issues when made into an update script in our deployment pipeline. A missing semicolon like this has caused data loss in production, so please take care.

Here is an example from the above referenced migration:
```C#
migrationBuilder.Sql(@"
    ALTER TABLE ""Vendor""
    RENAME COLUMN ""Id"" TO ""Code"";

    ALTER TABLE ""Site""
    RENAME COLUMN ""VendorId"" TO ""VendorCode"";
");
```